docs/en/observability/whats-new.asciidoc

```diff
@@ -11,4 +11,278 @@ https://www.elastic.co/blog/elastic-observability-7-6-0-released[7.6] |
 https://www.elastic.co/blog/elastic-observability-7-5-0-released[7.5] |
 https://www.elastic.co/blog/elastic-observability-update-7-4-0[7.4]
 
-coming[7.10.0]
+[discrete]
+== Give {minor-version} a try
+
+Try {minor-version} now by deploying {es} and {kib} on
+{ess-product}[Elastic Cloud] or
+by https://www.elastic.co/start[downloading them].
+
+// tag::710-new[]
```

Comment:
This looks good, but let's make it generic so it doesn't have to be updated each release on the stack-docs side.

```suggestion
// tag::whats-new[]
```